src/main/java/uk/gov/hmcts/reform/pcqbackend/service/SubmitAnswersService.java

```diff
@@ -190,8 +258,8 @@ public ProtectedCharacteristics getProtectedCharacteristicsById(String pcqId) {
             return handleInternalErrors(pcqId, coRelationId, ioe);
         }
 
-        return PcqUtils.generateResponseEntity(pcqId, HttpStatus.OK,
-                                                     environment.getProperty("api-error-messages.accepted"));
+        return PcqUtils.generateResponseEntity(pcqId, HttpStatus.CREATED,
```

Comment:
Code looks really good @priyankaVerma21 nice one. Only issue I can see and it's more me being picky is that the HttpStatus.CREATED wouldn't be true if an existing PCQ records was being updated. Can this be changed back to a HttpStatus.OK if updating? So you'll need to pass the correct HTTP Status depending on creating or updating.

Reply:
Thanks Dave for reviewing . I have changed this against 380 pull request which us related to PCQ-984.